src/sentry/static/sentry/app/views/organizationGroupDetails/eventToolbar.jsx

```diff
@@ -211,11 +205,11 @@ const GroupEventToolbar = createReactClass({
         </span>
       </div>
     );
-  },
-});
+  }
+}
 
-const RelatedTransactionButton = styled(Button)`
-  margin-right: ${space(2)};
+const NavigationButtons = styled(ButtonBar)`
+  float: right;
```

Comment:
NB: Is there an easy way to remove the float?

Reply:
Perhaps, but I think it would require changing the layout more of the surrounding elements as they are using floats too.